pkg/controllers/user/logging/config/config.go

```diff
@@ -0,0 +1,45 @@
+package constant
+
+const (
+	LoggingNamespace   = "cattle-logging"
+	ClusterLoggingName = "cluster-logging"
+	ProjectLoggingName = "project-logging"
+)
+
+//daemonset
+const (
+	FluentdName        = "fluentd"
+	FluentdHelperName  = "fluentd-helper"
+	FluentdHelperImage = "rancher/fluentd-helper:v0.0.1"
```

Comment:
@deniseschannon looks like we might need to implement privateRegistry/airgap support for helper images used by alerting/logging